### PR TITLE
Jj add user to trip

### PR DESCRIPTION
**Pull request by joaojusto**

### CODE REVIEW

File: client/views/users/sign_in.coffee [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Template.signIn.submit = () ->
+  email = $('#email').val()
+  password = $('#password').val()
+  Meteor.loginWithPassword(email, password,
```
Comment (gabrielpoca) [inline client/views/users/sign_in.coffee]: What do you think of this syntax instead?

```
Meteor.loginWithPassword email, password, (err) ->
  if err
    console.log(err)
  else
    Router.go 'tripsIndex'
```

File: client/views/users/sign_in.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<template name='signIn'>
+  {{#headingAlpha}}SignIn{{/headingAlpha}}
+  <p>SingIn</p>
+  <form>
+      <input id='email' type='email' name='email' placeholder='Type your email here...' class='TextInput'>
+      <input id='password' type='password' name='password' placeholder='Type your password here...' class='TextInput'>
```
Comment (gabrielpoca) [inline client/views/users/sign_in.html]: What do you think of using [javascript objects as arguments](https://www.discovermeteor.com/blog/spacebars-secrets-exploring-meteor-new-templating-engine)? I don't say for everything, things like class I think belong here.

File: client/views/users/sign_out.coffee [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Template.signOut.created = ()  ->
```
Comment (gabrielpoca) [inline client/views/users/sign_out.coffee]: You don't need the `()` here.

File: client/views/users/sign_in.coffee (hunk 1 above)
Comment (joaojusto) [inline client/views/users/sign_in.coffee]: It´s better

File: client/views/users/sign_in.coffee [hunk 4]
```diff
@@ -0,0 +1,14 @@
+Template.signIn.submit = () ->
+  email = $('#email').val()
+  password = $('#password').val()
+  Meteor.loginWithPassword(email, password, (err) ->
```
Comment (gabrielpoca) [inline client/views/users/sign_in.coffee:19]: You can remove the first `(`, like you did on the other function.

File: client/views/users/sign_up.html [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<template name='signUp'>
+  {{#headingAlpha}}SignUp{{/headingAlpha}}
+  <p>SingUp</p>
+  <form>
+      <input id='email' type='email' name='email' placeholder='Type your email here...' class='TextInput'>
```
Comment (gabrielpoca) [inline client/views/users/sign_up.html]: The indentation is wrong here.

File: lib/router.coffee [hunk 6]
```diff
@@ -1,8 +1,18 @@
 Router.configure
   layoutTemplate: 'layout'
 
-Router.onBeforeAction ->
-  NavigationVent.reset()
+beforeHooks = {
```
Comment (gabrielpoca) [inline lib/router.coffee]: you don't need the `{ }` here also

**Comment by gabrielpoca** [comment]
:+1:

**Pull request merged**